e2e-tests/portal/cypress.config.js

```diff
@@ -1,5 +1,12 @@
 const { defineConfig } = require('cypress');
 const { createTasks } = require('../support/tasks');
+const dotenv = require('dotenv');
+const path = require('path');
+
+// Read from root `./.env` directory
+dotenv.config({ path: path.resolve(__dirname, '../../.env') });
```

Comment:
Do we need this since `e2e-tests` has its own `.env`?
If must then I would suggest updating the whole `e2e-tests` to ready from root `.env` and remove `e2e-tests` `.env.sample`.